get/security.html

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+id: security
+title: Secure On-prem
+---
+
+<header>
+  <h1>Sandstorm is a secure open source server<br> you can run on-prem.</h1><p>Use great web apps while keeping ownership of your data.</p>
+  <a class="demo" href="http://demo.sandstorm.io">Try a Quick Demo</a><a class="get" href="https://oasis.sandstorm.io/shared/RvA7wR5iE6Jef0NYJTI_lVtnXyHjG2v3FR3o8FBs8dI">Get Sandstorm for Work</a>
+</header>
+
+<section id="why-sandstorm">
+  <h2>Why use Sandstorm?</h2>
+  <h3>Your employees deserve flexible online collaboration tools. You deserve the peace of mind that comes from owning your data. Apps installed on Sandstorm are secure by design.</h3>
+  
+  <ul>
+    <li class="why-onprem">
+      <h3><strong>On-prem deployment</strong></h3><p><strong>Run Sandstorm on hardware that you control.</strong>  When you put your data in an app's "cloud," you are trusting other companies with your security. Some have great security teams, but some don't, and it's hard to tell from the outside. Sandstorm lets you keep your data in-house, behind your firewall, where you are in control — but without the usual hassle of maintaining your own infrastructure. Read more about <a href="/business">Sandstorm for Work features</a>.</p>
+
+    <li class="why-finegrained">
+      <h3>Fine-grained isolation</h3><p><strong>Sandstorm isolates every document in its own secure container</strong>. That means that most security bugs in apps don't matter, because only people who already had access to a document can attack it. Serious security bugs are mitigated when an app runs on Sandstorm. To date, Sandstorm's hardened sandbox has proactively protected users against <a href="https://docs.sandstorm.io/en/latest/using/security-non-events/">39 documented security vulnerabilities</a>.</p>

```

Comment:
How about:
"To date, Sandstorm's hardened sandbox has proactively protected users against over 90% of documented security vulnerabilities we've researched."

Or "nearly all" instead of "over 90%"